Task 3

```diff
@@ -0,0 +1,55 @@
+function ACRemote(mode, state, temperature){
+  this.mode = mode;
+  this.temperature = temperature != undefined ? temperature : 26;
+  this.state = state != undefined ? state : false;
+  this.prevTemp = 26;
```

Comment:
These properties are accessible outside the function. Do I still need functions?